i18n/en.json

```diff
@@ -1815,6 +1815,11 @@
   "admin.reload.reloadDescription.featureName": "Reload Configuration from Disk",
   "admin.reload.reloadDescription.recycleDatabaseConnections": "Environment > Database > Recycle Database Connections",
   "admin.reload.reloadFail": "Reloading unsuccessful: {error}",
+  "admin.reporting.workspace_optimization.chip_problems": "Problems: {count}",
+  "admin.reporting.workspace_optimization.chip_suggestions": "Suggestions: {count}",
+  "admin.reporting.workspace_optimization.chip_warnings": "Warnings: {count}",
+  "admin.reporting.workspace_optimization.overall_workspace_score": "Overall Workspace Score",
+  "admin.reporting.workspace_optimization.overall_workspace_score_description": "Stay on top of how optimized your workspace is! This score is a total of the categories of key indicators of health and growth below. Ensure your workspace is in its most optimized state so you and your user base get the most out of Mattermost.",
```

Comment:
not related: we should port the other translations (currently under `workspaceOptimization.*`)

Reply:
ok